mos.yml

```diff
@@ -369,6 +369,52 @@ conds:
         - ["ssw2", "ssw", {title: "SSW2 settings"}]
         - ["ssw2.name", "Input 2"]
 
+  - when: build_vars.MODEL == "ShellyColorBulb"
+    apply:
+      name: color-bulb # To allow OTA from stock fw.
+      libs:
+        - origin: https://github.com/mongoose-os-libs/mongoose
+          variant: esp8266-nossl
+      build_vars:
+        FLASH_SIZE: 2097152
+        FS_SIZE: 262144
+        BOOT_CONFIG_ADDR: 0x1000  # To be compatible with stock firmware.
+        MGOS_ROOT_FS_TYPE: SPIFFS
+      cdefs:
+        PWM_SUPPORT: 1
+        PRODUCT_HW_REV: '"1.0"'
+        STOCK_FW_MODEL: '"SHCB-1"'
+        # We don't use SSL, HomeKit uses its own crypto. This saves ~120K.
+        MG_ENABLE_SSL: 0
+      config_schema:
+        - ["shelly.mode", 4] # 4 - RGBW"
+        - ["device.id", "shellycolorbulb-??????"]
+        - ["shelly.name", "shellycolorbulb-??????"]
+        - ["wifi.ap.ssid", "shellycolorbulb-??????"]
```

Comment:
I checked the stock firmware, here it is down case all other new once are camel case.